### ISSUE TITLE
get start date of study in API

### ISSUE THREAD

**Issue by hamilton** [mozilla/normandy#1404]
How difficult would it be to get a start date for a study added to the normandy API? I'm consuming the [`recipe`](https://normandy.cdn.mozilla.net/api/v2/recipe/) list for a Mozilla data project, and it'd be VERY useful for folks looking at the data to know when a study began.

**Comment by rehandalal** [comment]
This should be relatively easy once we fix #1363. Closing this in favor of that.

**Comment by rehandalal** [comment]
I should however point out that we will only have accurate data for recipes that were enabled after #1203 landed and was deployed.
